cmd/juju/controller/addmodel.go

```diff
@@ -55,20 +55,30 @@ type addModelCommand struct {
 
 const addModelHelpDoc = `
 Adding a model is typically done in order to run a specific workload.
-To add a model, you must at a minimum specify a model name. You may
-also supply model-specific configuration, a credential, and which
-cloud/region to deploy the model to. The cloud/region and credentials
-are the ones used to create any future resources within the model.
 
+To add a model, you must specify a model name. 
 Model names can be duplicated across controllers but must be unique for
 any given controller. Model names may only contain lowercase letters,
 digits and hyphens, and may not start with a hyphen.
 
-Credential names are specified either in the form "credential-name", or
-"credential-owner/credential-name". There is currently no way to acquire
-access to another user's credentials, so the only valid value for
-credential-owner is your own user name. This may change in a future
-release.
+To add a model, Juju requires a credential.
+If a controller has a credential for the user creating the model, 
+it will be used.
+If a controller does not have a credential, 
+this command allows to specify desired named credential with --credential option.
+This option stores provided named credential in the controller.
+
+Another way of ensuring that a named credential is stored in the controller, is
+by running update-credential command.
+
+Named credentials are defined in Juju via either 
```

Comment:
This sounds strange and vague. What does "defined in Juju" mean? What are "actual names", and what are other types of names if they are no actual names?

Suggested rewording:

> To add a credential for add-model, use one of the "juju add-credential" or "juju autoload-credentials" commands. These will add credentials to the Juju client, which "juju add-model" will upload to the controller as necessary.